web/modules/custom/joinup_user/joinup_user.install

```diff
@@ -51,6 +51,7 @@ function joinup_user_install() {
     'access content',
     'access tour',
     'access user profiles',
+    'never autoplay videos',
```

Comment:
Still don't understand why we're doing this crap. Why the permissions are not set via config management. That would make obsolete also the post-update.

Reply:
Hmm, I tested this again now. When we decided this approach it was that no matter how we tried it, we would get the "the config object already exists in active configuration" error. No matter if we were to place the files in the `joinup_user` module, or the `joinup` profile.
I see that now, we still cannot place the files in the `joinup_user` module but we can place them in the `joinup` profile. I will change this.